### DISCUSSION TITLE
Record the engine's applicationId in KyuubiSessionEvent.

### DISCUSSION THREAD

**Discussion by wForget** [apache/kyuubi#1609] [Ideas]
I want to record the applicationId of the engine in KyuubiSessionEvent, and then I will collect KyuubiServerEvent into elasticsearch so that I can quickly find the yarn application of the session.

How to get the applicationId in kyuubiServer？
1. If the new engine is created, we can get it in `ProcBuilder`.
2. If an existing engine is used, we can execute `select engine_id()` to get.

**Comment by yaooqinn** [comment]
Good idea, and we can put it in TOpenSessionResp for both cases

**Reply by wForget** [reply]
> Good idea, and we can put it in TOpenSessionResp for both cases

Thanks @yaooqinn , this is a good way, I will try it.
